src/main/java/com/afornalik/ox/MainApp.java

```diff
@@ -1,17 +1,55 @@
 package com.afornalik.ox;
 
 import com.afornalik.ox.board.Board;
+import com.afornalik.ox.board.FieldStatus;
+import com.afornalik.ox.board.OutOfBoardException;
+import com.afornalik.ox.progress.GameProgress;
 import com.afornalik.ox.ui.UI;
 import com.afornalik.ox.ui.UIConsole;
 import com.afornalik.ox.ui.UIDrawBoard;
 
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class MainApp {
 
     public static void main(String[] args) {
 
```

Comment:
I understand that current main() implementation is only for testing purposes?

Reply:
Yes exactly, just to play demo version.